src/test/handler.test.ts

```diff
@@ -66,30 +64,29 @@ const setDefaults = async () => {
 }
 
 describe('getParamsFromUriPath', () => {
-    beforeEach(setDefaults)
     it('extracts repo and file params from root folder', () =>
         expect(getParamsFromUriPath('git://github.com/sourcegraph/sourcegraph?264...#index.tsx')).toEqual({
-            repo: 'sourcegraph/sourcegraph',
+            repo: '/sourcegraph/sourcegraph',
             file: 'index.tsx',
         }))
 
     it('extracts repo and file params from subfolder', () =>
         expect(
             getParamsFromUriPath('git://github.com/sourcegraph/sourcegraph?264...#web/src/e2e/index.e2e.test.tsx')
         ).toEqual({
-            repo: 'sourcegraph/sourcegraph',
+            repo: '/sourcegraph/sourcegraph',
             file: 'web/src/e2e/index.e2e.test.tsx',
         }))
 
-    it('return empty repo if host is not GitHub', () =>
-        expect(getParamsFromUriPath('git://unknownhost.com/sourcegraph/testrepo#http/req/main.go')).toEqual({
-            repo: null,
-            file: 'http/req/main.go',
+    it('return null if URI is corupt', () =>
```

Comment:
```suggestion
    it('returns null if URI is corrupt', () =>
```

Reply:
oh shoot, I forgot to commit this change.